src/txkube/testing/_eliot.py

```diff
@@ -26,15 +26,13 @@ def _eliottree(logs):
 
     :return bytes: The rendered string.
```

Comment:
It doesn't appear to still return bytes.  This doesn't seem to be a necessary change and I'm not sure it's desirable.  Can you explain why returning text instead of bytes is better here?  And if it is, all the rest of the callers need to be fixed to deal with this.

Reply:
I've changed the docstring.  It looks like the new `render_tasks()` function returns unicode, while the deprecated `render_task_nodes()` function returns bytes.  So maybe the eliottree library is moving towards representing strings as unicode, and it it best to move the code here in this direction also.

The only caller of this I found was `CaptureEliotLogs`, and I don't see anything that needs to be changed there.

Reply:
Here's an example of "meaningful":

```
Failure: testtools.testresult.real._StringException: foo: {{{☃}}}
```

Reply:
@exarkun I'm still not fully understanding, can you please clarify?
`src/txkube/testing/_eliot.py` is a private module used by the tests for txkube.  
`src/txkube/testing/_testcase.py` imports _eliot.CaptureEliotLogs

`CaptureEliotLogs` is called from `AsynchronousDeferredRunTest._get_log_fixture`, which
is invoked from many tests such as the ones in `txkube.test.test_memory.KubernetesClientIntegrationTests`.

These tests pass on Python 2 and Python 3 with this patch.

Is there anything else to work on for this PR?